### ISSUE TITLE
Schedule not working after #519

### ISSUE THREAD

**Issue by mananwason** [fossasia/open-event-android#528]
After #519, we have changed the parsing of data and the schedule is now broken. Will have to update everything according to the new parser.

**Comment by the-dagger** [comment]
@mananwason We still don't have a schedule json from the orga-server right?

**Comment by mariobehling** [comment]
All info is already in the sample files for a long time. I really hope you guys can make this project happen until the end of GSoC. There are still issues in the sample files, but they are usable: https://github.com/fossasia/open-event/tree/master/sample/FOSSASIA16

**Comment by the-dagger** [comment]
@mariobehling Sure thanks.
But I believe, we need a separate json for `event-dates`, something like this
```{
  "dates": [
    {
      "date": "2015-05-05"
    },
    {
      "date": "2015-05-07"
    },
    {
      "date": "2015-09-16"
    }
  ]
}``` <br>so that we can easily add corresponding tabs for the schedule.
Iterating through the sessions.json for this is a quite challenging task.
I'll create an issue for this in the orga-server repo.
Also, I'm sure we will be able to finish this us until GSoC, the app generator is done and integrated with orga-server.
Only issue remaining is parsing the newer json files, which I and @mananwason are working on in #519 .
Cheers.

**Comment by mananwason** [comment]
@mariobehling @the-dagger There is no requirement of creating another issue. I have talked to @aviaryan in https://github.com/fossasia/open-event-orga-server/issues/1878. I'll fix this next. I was working on higher priority issues for now.

**Comment by the-dagger** [comment]
Ok, perfect. :+1:
